test/tools/fuzzTesting/fuzzHelper.h

```diff
@@ -47,14 +47,17 @@ class RandomCode
 	static RandomCodeBase& get()
 	{
 		static RandomCode instance;
-		return instance.generator;
+		return *instance.generator;
 	}
 	RandomCode(RandomCode const&) = delete;
 	void operator=(RandomCode const&) = delete;
 
 private:
-	RandomCode(){}
-	BoostRandomCode generator;
+	RandomCode()
+	{
+		generator.reset(new BoostRandomCode());
+	}
+	std::unique_ptr<RandomCodeBase> generator;
```

Comment:
Why is this not `RandomCodeBase generator;`?

Reply:
Because there will be another implementation other than BoostRandomCode

Reply:
OK.

Reply:
I still don't agree on this. Introduce this change when introducing the other implementation of RandomCodeBase.

Reply:
there was. during the refactoring of this PR we decided to remove it.

Reply:
So also remove this.